83 Display answered questions bug

Alkhimovmv opened the pull request:
I fix this problem, now everything is displayed correctly, @AlexKay28 please check

AlexKay28 commented:
@Alkhimovmv 
can u explain what you did shortly please?

alantishin commented on frontend/src/components/Question/Question.tsx:
```diff
@@ -61,14 +61,19 @@ const Question = () => {
+    }
     const session_token = window.localStorage.getItem('session_token')
-    api.patch('/answer', data, { headers: { session_token: `${session_token}`}})
+    await api.patch('/answer', data, { headers: { session_token: `${session_token}`}})
```
it looks very strange
you can
```
const response = await api.patch('/answer', data, { headers: { session_token: `${session_token}`}})
const answerData = {answer: response.data.answer}
setAnswerData(answerData)

const reponse2 = await api.post('/question', questionConfig, { headers: { session_token: `${session_token}` } })
setQuestion(reponse2.data.question)
...
```

When use async/await it exclude necessity of using then

Alkhimovmv replied:
Yes, you are absolutely right, I fixed it

Alkhimovmv commented:
I made the /question request run after the /answer request. Typed variables in "Question" and "Answer" components. Also changed the table in accordance with the layout, that is, removed the title of the columns in the rows with previous answers
